lua/lspconfig/server_configurations/tinymist.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  default_config = {
+    cmd = { 'tinymist' },
+    filetypes = { 'typst' },
+    root_dir = function()
```

Comment:
I aussme there can use single_file_support  and root can be use .git folder?

Reply:
no single_file_support,but i added the `.git` support